src/client/index.js

```diff
@@ -102,7 +102,7 @@ const app = {
 const env = process.env.APP;
 /* eslint-enable no-undef */
 const front = new Front("app", app, config, env);
-
+front.store.dispatch(appSetProperties(front.appProperties));
```

Comment:
Where come front.appProperties ? this call should be done in next front.start(true);

Reply:
appProperties is created by Front constructor by `this.appProperties = Front.combinePropsEnv(appProperties, appEnv);`